functions/nodejs-apps.bash

```diff
@@ -29,6 +29,7 @@ nodejs_setup() {
 
 frontail_setup() {
   nodejs_setup
+  return
```

Comment:
Why is this 'return' here?
Will 'frontail' work after this zram PR?

Reply:
Thanks for reminding me. The frontail build is currently broken when running against our new automated testing image, that's why I skipped it and forgot later on.
But we mustn't fix the frontail install but rather the test pipeline.

Reply:
removed bad 'return'